diff --git a/dist/cjs/client/stdio.js b/dist/cjs/client/stdio.js
index ee48b40d..cd24a0f3 100644
--- a/dist/cjs/client/stdio.js
+++ b/dist/cjs/client/stdio.js
@@ -77,7 +77,7 @@ class StdioClientTransport {
                 },
                 stdio: ['pipe', 'pipe', this._serverParams.stderr ?? 'inherit'],
                 shell: false,
-                windowsHide: node_process_1.default.platform === 'win32' && isElectron(),
+                windowsHide: node_process_1.default.platform === 'win32',
                 cwd: this._serverParams.cwd
             });
             this._process.on('error', error => {
diff --git a/dist/esm/client/stdio.js b/dist/esm/client/stdio.js
index e1e4b9c6..6b2d030e 100644
--- a/dist/esm/client/stdio.js
+++ b/dist/esm/client/stdio.js
@@ -70,7 +70,7 @@ export class StdioClientTransport {
                 },
                 stdio: ['pipe', 'pipe', this._serverParams.stderr ?? 'inherit'],
                 shell: false,
-                windowsHide: process.platform === 'win32' && isElectron(),
+                windowsHide: process.platform === 'win32',
                 cwd: this._serverParams.cwd
             });
             this._process.on('error', error => {
